Figures/Figures.cs

```diff
@@ -1,24 +1,50 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
 namespace Figures
 {
     public abstract class Figure: IFigure
     {
+#region properties
+		
         public int ID { get; set; }
-        public double XCoordinate { get; set; }
-        public double YCoordinate { get; set; }
-        public double Angle { get; set; }
+        protected double centerX;
+        protected double centerY;
 
-        public abstract void RotateClockwize(double angle);
+        protected Orientation orientation;
+        
+        public double CenterX
```

Comment:
Instead of having ptotected 'centerX' field and public 'CenterX' property w/o setter, you could simply define the automatic property with protected setter:

public CenterX { get; protected set; }